src/configuration.rs

```diff
@@ -7,14 +7,11 @@ use std::sync::Arc;
 use crate::configuration::action_set::ActionSet;
 use crate::configuration::action_set_index::ActionSetIndex;
 use crate::data;
+use crate::data::Predicate;
 use crate::data::PropertyPath;
```

Comment:
Can't this be removed too and get us the same benefits (with the additional same changes to `data/mod.rs` and making the struct `pub(super)`) as `get_property`?

Reply:
not sure I follow. `use crate::data::PropertyPath;` cannot be removed. It is being used in this module as part of the `SelectorItem`.